apm-agent-common/src/main/java/co/elastic/apm/agent/util/Version.java

```diff
@@ -70,7 +74,7 @@ public int compareTo(Version another) {
                 return left < right ? -1 : 1;
             }
         }
-        return 0;
+        return another.suffix.compareTo(suffix);
```

Comment:
I don't think this is something to rely on.
At most, consider one that `hasSuffix()` as lower than one that doesn't, but even that is not so solid.